glusterfsd/src/glusterfsd-mgmt.c

```diff
@@ -325,6 +330,28 @@ glusterfs_handle_terminate(rpcsvc_request_t *req)
     }
     ctx = glusterfsd_ctx;
 
+    dict = dict_new();
+    if (!dict) {
+        return -1;
+    }
+    if (xlator_req.dict.dict_len) {
+        ret = dict_unserialize(xlator_req.dict.dict_val,
+                               xlator_req.dict.dict_len, &dict);
+        if (ret < 0) {
+            gf_log(this->name, GF_LOG_ERROR,
+                   "Failed to unserialize "
+                   "req-buffer to dictionary");
+            goto err;
+        }
+    }
+
+    ret = dict_get_str(dict, GLUSTER_BRICK_GRACEFUL_CLEANUP, &value);
+    if (!ret) {
+        ret = gf_string2boolean(value, &graceful_cleanup);
+        if (ret)
+            graceful_cleanup = _gf_false;
```

Comment:
When is `graceful_cleanup` set to `_gf_true`?

Reply:
when a key (GLUSTER_BRICK_GRACEFUL_CLEANUP) is true then graceful_cleanup will be true.

Reply:
I have executed regression test-suite(https://github.com/gluster/glusterfs/pull/1892) after enabling graceful_key, it is working fine. After done regress testing we will enable the key as a default option.